packages/react/src/context.tsx

```diff
@@ -0,0 +1,16 @@
+import LogtoClient from '@logto/browser';
+import { createContext } from 'react';
+
+export type LogtoContextProps = {
+  logtoClient?: LogtoClient;
+  loadingStackCount: number;
+  setLoadingStackCount: React.Dispatch<React.SetStateAction<number>>;
+};
+
+export const LogtoContext = createContext<LogtoContextProps>({
+  logtoClient: undefined,
+  loadingStackCount: 0,
+  setLoadingStackCount: (): never => {
+    throw new Error('Must be used inside <LogtoProvider>');
```

Comment:
we can define the error builder in this file and reuse in `hooks`:

```ts
const throwContextError = () => throw new Error('...');
```

Reply:
[LOG-1854](https://linear.app/silverhand/issue/LOG-1854/declare-a-standardized-error-class-in-react-context)
Ticket created. Will do in a follow-up PR